pyccel/ast/basic.py

```diff
@@ -42,3 +42,11 @@ def precision(self):
     @property
     def order(self):
         return self._order
+
+    def copy(self, x):
```

Comment:
Since the name `copy` is used in many Python libraries (including Numpy) to identify functions and methods that create a new instance, I recommend to change the name of this method, e.g. to `copy_attributes`.

Reply:
okay